src/lib/hooks/use-best-quote/utils/available.ts

```diff
@@ -0,0 +1,59 @@
+import {
```

Comment:
Moving these to `use-best-quote` to be more local (than global) - as they are just used in these hooks